src/pages/Search/SearchHoldReasonPage.tsx

```diff
@@ -18,11 +19,18 @@ function SearchHoldReasonPage({route}: PlatformStackScreenProps<Omit<SearchRepor
     const {translate} = useLocalize();
     const {backTo = '', reportID} = route.params ?? {};
     const context = useSearchContext();
+    const [reports] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}`, {canBeMissing: false});
+    const [reportActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, {canBeMissing: false});
```

Comment:
Same here. We're connecting to all reportActions for that report, but we only care about the IOU reportActions. Can we add an Onyx selector to only subscribe to those actions please? Here's an example https://github.com/Expensify/App/blob/24ad9568521cbf731cbfb3bf70c38ab1c1d9a9b2/src/components/TransactionItemRow/DataCells/ChatBubbleCell.tsx#L29-L32